net/src/ipc_net_worker.rs

```diff
@@ -119,13 +149,91 @@ impl IpcNetWorker {
                 let out: Box<NetWorker> = Box::new(IpcClient::new(h, socket, block_connect)?);
                 Ok(out)
             }),
+            None,
         )
     }
 
     // -- private -- //
 
+    /// create a new IpcNetWorker instance pointing to a process that we spawn here
+    fn priv_spawn(
+        handler: NetHandler,
+        cmd: String,
+        args: Vec<String>,
+        work_dir: String,
+        env: HashMap<String, String>,
+        block_connect: bool,
+    ) -> NetResult<Self> {
+        let mut proc = std::process::Command::new(cmd);
+        proc.stdout(std::process::Stdio::piped());
+        proc.args(&args);
+        proc.envs(&env);
+        proc.current_dir(work_dir);
+        let mut proc = proc.spawn()?;
+
+        let re_ready = regex::Regex::new("#IPC-READY#")?;
+        let re_ipc = regex::Regex::new("(?m)^#IPC-BINDING#:(.+)$")?;
+        let re_p2p = regex::Regex::new("(?m)^#P2P-BINDING#:(.+)$")?;
+
+        let mut ipc_binding = "".to_string();
```

Comment:
If you need an empty string, always use `String::new()`, which gives you one

Reply:
done